lib/vdsm/storage/lvm.py

```diff
@@ -1439,6 +1439,26 @@ def createVG(vgName, devices, initialTag, metadataSize, force=False):
         _lvminfo._invalidatevgs(vgName)
         log.debug("Cache after createvg %s", _lvminfo._vgs)
 
+    # Disable autoactivation after creating a VG
+    _disable_autoactivation_vg(vgName)
+
+
+def _disable_autoactivation_vg(vg_name):
+    """Avoid undesired VG activation by disabling autoactivation attribute
+    on a given VG.
+
+    Args:
+        vg_name (str): Name of the VG
+
+    Raises:
+        se.VolumeGroupCreateError: LVM error occurred
+    """
+    cmd = ["vgchange", "--setautoactivation", "n", vg_name]
+    try:
+        _lvminfo.run_command(cmd)
```

Comment:
We need to use the vg devices in this command, so lvm does not try to look at other PVs. See how other commands use devices=.